environment/environment_templates.go

```diff
@@ -239,10 +242,21 @@ func NewChainlinkClusterForAlertsTesting(nodeCount int) K8sEnvSpecInit {
 		cManifest.id = fmt.Sprintf("%s-%d", cManifest.id, i)
 		chainlinkGroup.manifests = append(chainlinkGroup.manifests, cManifest)
 	}
+
+	dependencyGroups := []*K8sManifestGroup{}
+
+	kafkaDependecyGroup := &K8sManifestGroup{
+		id:        "KafkaGroup",
+		manifests: []K8sEnvResource{NewKafkaHelmChart()},
+	}
+
 	dependencyGroup := getBasicDependencyGroup()
-	addPostgresDbsToDependencyGroup(dependencyGroup, nodeCount)
 	addServicesForTestingAlertsToDependencyGroup(dependencyGroup, nodeCount)
-	return addNetworkManifestToDependencyGroup("basic-chainlink", dependencyGroup, chainlinkGroup)
+	addPostgresDbsToDependencyGroup(dependencyGroup, nodeCount)
+
+	dependencyGroups = append(dependencyGroups, kafkaDependecyGroup)
```

Comment:
You can get rid of the multiple appends by just adding them in via the declaration `dependencyGroups := []*K8sManifestGroup{kafkaDependecyGroup, dependencyGroup}`